hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -131,8 +131,9 @@ const (
 	controlPlanePKIOperatorSignsCSRsLabel                      = "io.openshift.hypershift.control-plane-pki-operator-signs-csrs"
 	useRestrictedPodSecurityLabel                              = "io.openshift.hypershift.restricted-psa"
 
-	etcdEncKeyPostfix    = "-etcd-encryption-key"
-	managedServiceEnvVar = "MANAGED_SERVICE"
+	etcdEncKeyPostfix                     = "-etcd-encryption-key"
+	managedServiceEnvVar                  = "MANAGED_SERVICE"
+	aroHCPKeyVaultManagedIdentityClientID = "ARO_HCP_KEY_VAULT_USER_CLIENT_ID"
```

Comment:
given that this will in turn be used by the CPO, it should go in https://github.com/openshift/hypershift/blob/main/support/config/constants.go

Reply:
and used by the CLI as well